### ISSUE TITLE
Logs TypeError on new version 1.3.2

### ISSUE THREAD

**Issue by jmendiola222** [MauriceButler/bunyan-loggly#23]
I got this error while updating to from `1.3.1` to `1.3.2`. Previous versions worked perfectly.

```
throw new TypeError('"string" must be a string, Buffer, or ArrayBuffer');
    ^
TypeError: "string" must be a string, Buffer, or ArrayBuffer
    at Function.byteLength (buffer.js:477:11)
    at exports.Loggly.Loggly.log (/.../node_modules/node-loggly-bulk/lib/loggly/client.js:161:32)
    at Bunyan2Loggly._processBuffer (/.../node_modules/bunyan-loggly/index.js:48:32)
    at Bunyan2Loggly._checkBuffer (/.../node_modules/bunyan-loggly/index.js:61:21)
    at Bunyan2Loggly.write (/.../node_modules/bunyan-loggly/index.js:37:10)
    at Logger._emit (/.../node_modules/bunyan/lib/bunyan.js:923:22)
    at Logger.info (/.../node_modules/bunyan/lib/bunyan.js:1045:24)
```

I'm using node `v8.4.0` and this packages dependencies:
```
"dependencies": {
    "bunyan": "1.8.12",
    "bunyan-loggly": "1.3.1",
    "bunyan-prettystream": "0.1.3",
    "config": "1.26.2",
    "jsonschema": "1.2.0",
    "mongodb": "2.2.31",
    "restify": "5.2.0",
    "restify-clients": "1.5.0"
  },
```

**Comment by jingram-classy** [comment]
https://github.com/loggly/node-loggly-bulk/issues/10

**Comment by tinder-dyakobian** [comment]
bump

**Comment by Shwetajain148** [comment]
Hi @jmendiola222 @jingram-classy, we have published a beta version of `node-loggly-bulk` library which will resolve above error. Please use beta package and let me know your feedback.

Install beta package using the below command: 
`npm install node-loggly-bulk@next`
or
`npm install node-loggly-bulk@2.2.1-beta`

CC: @mostlyjason

**Comment by kslonka** [comment]
bump

**Comment by mostlyjason** [comment]
@MauriceButler we'd like to make this new version available to customers this week. If you or anyone else wants to test the beta, please do so by Wednesday.

**Comment by MauriceButler** [comment]
@Shwetajain148  @mostlyjason I have tested that beta version and it resolves this issue.

As soon as you make that available I will get it integrated and a new version published.

This will also allow me to resolve #24

**Comment by Shwetajain148** [comment]
Hi @MauriceButler, thanks for confirming that our beta package worked fine for you as well.

I have published the new npm package `version 2.2.1` of `node-loggly-bulk` so now you can also update your library. 

cc: @mostlyjason

**Comment by MauriceButler** [comment]
Thanks @Shwetajain148 

updated with 2b44f10760f8c24be297597d252c240e2fbf74d0

Version 1.3.3 now published
